packages/extension-ui/src/Popup/Signing/index.tsx

```diff
@@ -2,27 +2,36 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import React, { useContext } from 'react';
+import React, { useContext, useState, useEffect } from 'react';
 
-import { Header, SigningContext } from '../../components';
+import { Header, Loading, SigningContext } from '../../components';
 import Request from './Request';
+import TransactionIndex from './TransactionIndex';
 
 export default function Signing (): React.ReactElement<{}> {
   const requests = useContext(SigningContext);
+  const [rqId, setRqId] = useState(0);
```

Comment:
No need for the abbreviation
```suggestion
  const [requestId, setRequestId] = useState(0);
```

Reply:
And is it `id` actually? Maybe `index`?